internal/logger_test.go

```diff
@@ -15,40 +15,199 @@
 package containersuseconnect
 
 import (
+	"bytes"
+	"fmt"
+	"log"
 	"os"
+	"path/filepath"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
-// The helper for testing the logger setup where:
-//
-//   - env: the value for the `logEnv` environment value.
-//   - expected: the expected path of the logger.
-//   - cleanup: whether the file has to be closed and removed.
-func testLogger(t *testing.T, env, expected string, cleanup bool) {
-	os.Setenv(logEnv, env)
-	f := GetLoggerFile()
-	if f.Name() != expected {
-		t.Fatalf("Wrong file")
-	}
+func TestIsValidLogPathFromRegularFile(t *testing.T) {
+	tempFile, err := os.CreateTemp("", "prefix")
+	assert.Nil(t, err)
```

Comment:
`require` is just a wrapper around `assert` that calls `t.FailNow`. I don't see any value in this change to be honest. 

Am I missing something?

Reply:
If the creation of the temporary directory fails, then there's really no value in continuing to execute the test, as it is already clear that it will fail too. That was my rationale, why require should be used here

Reply:
I'll make a few tests, but for now I'll leave it like this. My idea is to rework other tests to use this library as it makes it easier to test and clearer to understand.